asteroid/models/base_models.py

```diff
@@ -68,16 +70,29 @@ def file_separate(self, *args, **kwargs):
         """Convenience for ``asteroid.separate.file_separate(self, ...)``."""
         return separate.file_separate(self, *args, **kwargs)
 
-    def _separate(self, wav, *args, **kwargs):
-        """Hidden separation method
+    def forward_wav(self, wav, *args, **kwargs):
+        """Separation method for waveforms.
+        In case the network's `forward` doesn't have waveforms as input/output,
```

Comment:
```suggestion
        """Separation method for waveforms.

        In case the network's `forward` doesn't have waveforms as input/output,
```

Reply:
We should document the expected return shape

Reply:
Added whitespace so that Sphinx makes a line break, assuming that was your intention?

Reply:
Yes, thanks.